.github/workflows/gg-shield-action.yaml

```diff
@@ -0,0 +1,38 @@
+name: GitGuardian scan
+
+on: 
+  push:
+  pull_request:
+    # Publish `main` as Docker `latest` image.
+    branches:
+      - main
+      - haruharu720test
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+
+jobs:
+  scanning:
+    name: GitGuardian scan
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.4.0
+        with:
+          fetch-depth: 0 # fetch all history so multiple commits can be scanned
+        env:
+          GITHUB_TOKEN: "87028c11-8bef-4bd6-b440-5d3851314ead"
```

Comment:
@ruzell22 Please remove this and make sure to invalidate your token because this one now has been exposed.